assets/src/components/ladderPage.tsx

```diff
@@ -210,6 +242,14 @@ const LadderPageWithTabs = (): ReactElement<HTMLDivElement> => {
               tab={routeTab}
               selectTab={() => dispatch(selectRouteTab(routeTab.uuid))}
               closeTab={() => dispatch(closeRouteTab(routeTab.uuid))}
+              showSaveIcon={!isPreset(routeTab) || isEditedPreset(routeTab)}
+              saveTab={() => {
+                if (isEditedPreset(routeTab)) {
+                  dispatch(savePreset(routeTab.uuid))
+                } else if (!isPreset(routeTab)) {
+                  dispatch(createPreset(routeTab.uuid, "Preset name"))
```

Comment:
**question:** It seems like all presets are currently named, "Preset name". Is that a known issue that will be addressed later?

Reply:
Yes - [this ticket](https://app.asana.com/0/1200180014510248/1201286872028647/f) will implement modals to confirm deletion of a preset as well as allowing the user to input the name to save a preset under. This also means these callbacks are going to have to change a bit - we're going to have to dispatch a change to the state that triggers the modal to open that itself takes a callback to invoke once the user has supplied a name and clicked "save."